pkg/storaged/client.js

```diff
@@ -206,6 +214,66 @@ function is_multipath_master(block) {
     return false;
 }
 
+export async function btrfs_poll() {
+    const usage_regex = /used\s+(?<used>\d+)\s+path\s+(?<device>[\w/]+)/;
+    if (!client.uuids_btrfs_subvols)
+        client.uuids_btrfs_subvols = { };
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231205-110900-7b3be2c9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)